firebase-appdistribution/src/main/res/layout/activity_feedback.xml

```diff
@@ -100,25 +100,35 @@
                 android:background="@drawable/card_background"
                 android:orientation="vertical">
 
-                <CheckBox
-                    android:id="@+id/screenshotCheckBox"
-                    android:layout_width="wrap_content"
+                <RelativeLayout
```

Comment:
Since there are only two child elements and their relative positioning is pretty straightforward, wouldn't it make sense to use a simpler option for this container, e.g. `LinearLayout` (horizontal)?

Reply:
I actually tried a `LinearLayout` before, but it was clear to me how I could get the checkbox to align to the left and the button to align to the right. With `RelativeLayout` you get the nice `android:layout_*` attributes which makes it easier.

Reply:
`RelativeLayout` looks good to me. Alternatively if you wanted to make it work with `LinearLayout` you might be able to add an empty `View` with a weight of 1 in between the two, as a spacer: https://developer.android.com/develop/ui/views/layout/linear#Weight

Reply:
`LinearLayout` with a spacer seems jankier, so I'd lean towards keeping it as a `RelativeLayout`.

Reply:
Or maybe you can use `gravity` instead of an empty `View`: 

> A [LinearLayout](https://developer.android.com/reference/android/widget/LinearLayout) respects [...] the gravity (right, center, or left alignment) of each child.

Reply:
Tried this, and it wasn't working. I'm probably not using `gravity` correctly here, but regardless I think the `RelativeLayout` is sufficient here and I like the explicitness of the `android:layout_*` attributes.
![Screenshot 2023-01-26 at 2 04 57 PM](https://user-images.githubusercontent.com/401051/214926900-147d706b-8310-4851-bee6-ef37d9375a93.png)